### ISSUE TITLE
linksafe should add the column to the output

### ISSUE THREAD

**Issue by koppor** [TechWiz-3/linksafe#11]
Limit to 10 failed links

Similar to [reviewdog action](https://github.com/reviewdog/action-setup) or https://github.com/errata-ai/vale-action

![grafik](https://user-images.githubusercontent.com/1366654/198397677-fcd33139-c8af-4ed6-b7bc-1167ee452708.png)

**Comment by TechWiz-3** [comment]
So the workflow will make a PR removing broken links?

**Comment by koppor** [comment]
The use-case is that there might be external. They do not have write-request to the repository. They use the feature of GitHub's pull request to contribute. For GitHub pull requests, some "fancy" interaction possibilities exist. I like the one for commenting on the concrete lines.

**Comment by koppor** [comment]
Maintainers of a repository still can decide whether they want to use the PR method or just commit to the `main` branch. I think, it really depends on the repository. For instance, at JabRef, we have >400 external contributors, whereas in my own "toy projects", I am the single one contributor and work differently.

**Comment by koppor** [comment]
TL;DR: linksafe should add the column to the output

---

I think, what would help there is to use [`errorformat`](https://github.com/reviewdog/errorformat) for output. (`{file}:{line number}:{column number}: {message}`)

Current output is:

```
==== Failed links ====
In ./README.md on line 113, link: https://github.com/search?utf8=%E2%9C%93&q=label%3Ahacktoberfest+state%3Aopen+type%3Aissue+language%3ADart&type=Issues&ref=advsearch&l=React&l=
In ./README.md on line 125, link: https://github.com/search?utf8=%E2%9C%93&q=label%3Ahacktoberfest+state%3Aopen+type%3Aissue+language%3ATypescript&type=Issues&ref=advsearch&l=Typescript
```

Having errorformat in place, one could use other tools for commenting on PRs (such as reviewdog).

---

One could also configure reviewdog to follow the format of `linksave`

    reviewdog -efm="In %f on line %l, %m" -diff="git diff FETCH_HEAD"

I need to test whether the column is a required input.

Independent of that, **linksafe should add the column to the output**

---

For direct posting: Google results:

- Stackoverflow question: https://stackoverflow.com/q/41662127/873282
- Project posting comments: https://github.com/harryi3t/lint-github-pr
